Azure/azure-cli
# [Storage] Fix #18352: `az storage fs file list --exclude-dir` breaks with `--show-next-marker`

**Description**

Fix #18352 

**Testing Guide**

`az storage fs file list --file-system --account-name --show-next-marker --exclude-dir`


---

This checklist is used to make sure that common guidelines for a pull request are followed.
